Sophmores_FinalProj/Game/Gameplay.cs

```diff
@@ -509,7 +511,14 @@ private static void Main()
                               "now opens! We are going into the final door!");
             Console.ResetColor();
             DoorStage4(p1, Enemies[10], Enemies[11]);
-            break;
+            if (p1.currentStage == 4)
+            {
+                break;
+            }
+            else
```

Comment:
@mkhanbas 

I don't see anything being skipped so no need for continue i don't think?